qiskit/providers/aer/noise/errors/standard_errors.py

```diff
@@ -231,6 +232,7 @@ def single_pauli(pauli):
             return {'name': 'y'}
         if pauli == 'Z':
             return {'name': 'z'}
+        return None
```

Comment:
This should match the `single_pauli` function in line 175 which has `raise NoiseError("Invalid Pauli string.")` if it reaches the end rather than return None.